src/components/Auth/SignUpForm.js

```diff
@@ -36,102 +32,106 @@ const SignUpForm = ({ toggleActiveForm }) => {
         setErrorMessage(Object.values(err.response.data).join(''));
       });
   };
+
   if (isLoggedIn) {
     return <Redirect to={referer} />;
   }
+
+  if (auth && auth.authTokens) {
+    return <p>Welcome!</p>;
+  }
+
   return (
-    <>
-      {(auth && auth.authTokens) || isLoggedIn ? (
-        <p>Welcome!</p>
-      ) : (
-        <>
+    <Box
+      component={Form}
+      display="flex"
+      flexWrap="wrap"
+      noValidate
+      autoComplete="off"
+      onSubmit={onSubmit}
+      data-testid="signupForm"
+      validationResolver={validationResolver}
+      defaultValues={defaultValue}
+    >
+      <Box component="h1" fontSize={18}>
```

Comment:
we should find a way to handle styles like these outside of components

Reply:
Yes, agree here.

Reply:
I agree, but so far we use the same style elsewhere in the codebase -- e.g. ``` <Typography variant="body2" color="textSecondary" component="p">``` (copied directly from material-UI examples). Perhaps our standard for CSS/some CSS refactoring could be set in a new PR, since does the trick and this PR is less about CSS standards.